src/api.ts

```diff
@@ -1,23 +1,29 @@
 import { Hono } from "hono";
 import _ from "lodash";
+import { Field } from "./game";
 
 export const api = new Hono();
 
 api.get("/field", async (c) => {
 	const redis = c.get("redis");
 	const field = await redis.get("field");
-	return c.json({ field });
+	return c.json({ result: field });
 });
 
 api.get("/field/seed", async (c) => {
 	const redis = c.get("redis");
-	await redis.set("field", Buffer.from(_.times(5000, () => _.random(9))));
+	const bitfield = redis.bitfield("field");
+	await redis.del("field");
+	for (const [index, value] of new Field().data.entries()) {
```

Comment:
It would be better to format the data into a string or buffer and then set with a single query. Unless upstash works differently.

Reply:
this works as expected, (I've tested it and it sends a single query with the bitfield command I created)
but I have some ideas for the architecture that may upend this I want to run by you